metrics/replicacpumem.go

```diff
@@ -0,0 +1,84 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/relab/hotstuff/metrics/types"
+	"github.com/relab/hotstuff/modules"
+	"github.com/shirou/gopsutil/v3/cpu"
+	"github.com/shirou/gopsutil/v3/mem"
+)
+
+// CPUMem metics measures the percentage of cpu and memory utilization on the node.
```

Comment:
This is kind of important info for a user. Please move this doc to the `metrics` package level, and integrate it with other documentation for the other metrics supported.

Note: Some typos in this text: `metic(s)` -> `metric(s)`